src/main/java/racinggame/domain/Car.java

```diff
@@ -1,36 +1,40 @@
 package racinggame.domain;
 
+import racinggame.domain.value.Name;
 import racinggame.domain.value.Position;
 
 import java.util.Objects;
 
 public class Car {
+    private static final int ADVANCED_CONDITION = 4;
 
-    private final String name;
+    private final Name name;
     private final Position position;
 
-    public Car(String name, Position position) {
+    public Car(Name name, Position position) {
         this.position = position;
         this.name = name;
-
     }
 
     public Car move(int moveValue) {
-        position.increase(moveValue);
+        if(validateMoveValue(moveValue)) {
```

Comment:
```suggestion
        if (validateMoveValue(moveValue)) {
```
코드 포맷팅 한 번 더 확인 부탁드리겠습니다.